nservicebus/msmq/distributor/configuration_master_core_[,4).partial.md

```diff
@@ -0,0 +1,5 @@
+It is also possible to use the `NServiceBus.Master` profile to start a Distributor on the endpoint with a Worker on its endpoint. To start the endpoint as a Master run the host from the command line, as follows:
```

Comment:
@weralabaj 
Distributor == a thing that distributes messages to workers
Worker == a thing that processes messages
Distributor *profile* == a profile that instructs NSB host to run Distributor only
Master *profile* == a profile that instructs NSB host to run Distributor and Worker

I am not sure how can I make this more clear :( Maybe mention in the beginning of the document that an instance of NSB endpoint can run a Worker, a Distributor or both, depending on configuration?